Z/zbc2014/build_tarballs.jl

```diff
@@ -0,0 +1,26 @@
+using BinaryBuilder
+
+name = "libzbc2014"
```

Comment:
The name of the directory must match exactly this name.  Either change this or move the directory as appropriate.  See also https://docs.binarybuilder.org/stable/building/#Name